Makefile

```diff
@@ -0,0 +1,50 @@
+# Package related
+TOOLSDIR=$(CURDIR)/hack/tools/bin
+BUILDDIR=$(CURDIR)/build/_output
+
+
+# If gobin not set, create one on ./build and add to path.
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(BUILD_DIR)/gobin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+export PATH:=$(GOBIN):${PATH}
+
+$(TOOLSDIR):
+	@mkdir -p $@
+
+$(BUILDDIR): ; $(info Creating build directory...)
+	mkdir -p $@
+
+# Paths to download the helm chart to.
+HELM_CHART_DEP_ROOT ?= $(BUILDDIR)/helmcharts
+# Helm chart version and url
+HELM_CHART_VERSION ?= 24.4.1
+NGC_HELM_CHART_URL ?= https://helm.ngc.nvidia.com/nvidia/charts
+
+build-docs: helm-docs
+	./repo.sh docs
+
+# helm-docs is used to generate helm chart documentation
+HELM_DOCS_PKG := github.com/norwoodj/helm-docs/cmd/helm-docs
+HELM_DOCS_VER := v1.14.2
+HELM_DOCS_BIN := helm-docs
+HELM_DOCS_PATH = $(abspath $(TOOLSDIR)/$(HELM_DOCS_BIN))
+HELM_DOCS = $(abspath $(TOOLSDIR)/$(HELM_DOCS_BIN))-$(HELM_DOCS_VER)
+$(HELM_DOCS): | $(TOOLSDIR)
+	$(call go-install-tool,$(HELM_DOCS_PKG),$(HELM_DOCS_BIN),$(HELM_DOCS_VER))
+	$Q ln $(HELM_DOCS) $(HELM_DOCS_PATH)
+
+# go-install-tool will 'go install' a go module $1 with version $3 and install it with the name $2-$3 to $TOOLSDIR.
+define go-install-tool
+	$Q echo "Installing $(2)-$(3) to $(TOOLSDIR)"
+	$Q GOBIN=$(TOOLSDIR) go install $(1)@$(3)
+	$Q mv $(TOOLSDIR)/$(2) $(TOOLSDIR)/$(2)-$(3)
+endef
+
+# Generate helm chart documentation in a reStructuredText format.
+helm-docs: $(HELM_DOCS)
+	mkdir -p ${HELM_CHART_DEP_ROOT} && cd ${HELM_CHART_DEP_ROOT} && curl -sL ${NGC_HELM_CHART_URL}/network-operator-${HELM_CHART_VERSION}.tgz | tar xz
```

Comment:
Can we have an option to run it on a directory in addition to the option of downloading the chart?

Reply:
I have added separate target for branch or ngc.